src/Modal/Modal.svelte

```diff
@@ -10,6 +10,10 @@
   export let borderColor = "";
   export let hideCloseButton = false;
 
+  // by default, all modals have padding
+  // use <Modal noPadding> ... </Modal> to remove
+  export let noPadding=false;
```

Comment:
Would the inline `style` prop be easier/more flexible here?

`<Modal style="padding: 0;" />`

Reply:
yes, makes more sense actually

Reply:
updated to your suggestion